src/tests/ftest/soak/harassers.yaml

```diff
@@ -197,7 +197,7 @@ mdtest_harasser:
     - 0
   dfs_oclass:
     - ["EC_2P1G1", "RP_2G1"]
-    - ["EC_4P2GX", "RP_3G1"]
+    - ["EC_4P2G1", "RP_3G1"]
```

Comment:
Curious - this runs mdtest with RP_2G1 for directories? FWIW, it's recommended to use GX on directories for performance reasons.